### PR TITLE
Disable Lottie animation on canvas

### PR DESCRIPTION
**Pull request by k15a**
This will disable the Lottie animations on the Framer canvas.

### CODE REVIEW

File: Lottie.framerfx/code/Lottie.tsx [hunk 1]
```diff
@@ -165,23 +174,29 @@ export class Lottie extends React.Component<Partial<Props>> {
...
         const options = {
             ...defaultOptions,
+            autoplay: !isStatic,
             animationData,
             loop,
         }
+
         return (
-            <Lottie
+            <ReactLottie
```
Comment (k15a) [inline Lottie.framerfx/code/Lottie.tsx:196]: How was this ever working?

File: Lottie.framerfx/code/Lottie.tsx [hunk 2]
```diff
@@ -165,23 +174,29 @@ export class Lottie extends React.Component<Partial<Props>> {
...
+
         if (!animationData) {
             return (
                 <div style={style}>
                     <img src={lottiePlaceholder} />
                 </div>
             )
         }
+
+        const isStatic = RenderTarget.hasRestrictions()
```
Comment (k15a) [inline Lottie.framerfx/code/Lottie.tsx:186]: Not sure if we want to make this controllable like we do with the Video component.

Comment (huntercaron) [reply Lottie.framerfx/code/Lottie.tsx:186]: TIL this function even exists. Im fine with preventing animations on the canvas for now

**Review by huntercaron — APPROVED** [review]
Nice! Ill rewrite / move this to modules soon but seems good for now

**Pull request merged**